invenio_rdm_records/fixtures/demo.py

```diff
@@ -31,27 +31,38 @@ class CachedVocabularies:
     """
 
     _resource_type_ids = []
+    _language_ids = []
+
+    @staticmethod
+    def _get_vocab(vocabulary):
+        """Get all records of a vocabulary."""
+        dir_ = Path(__file__).parent
+
+        return VocabulariesFixture(
+            system_identity,
+            [Path("./app_data"), dir_ / "data"],
+            "vocabularies.yaml",
+        ).get_records_by_vocabulary(vocabulary)
```

Comment:
minor: Now that we are considering multiple vocabularies and we have dealt with importing them, it might be that we want to rely on the vocabularies services instead: `vocabularies_service.read_many` might give us a pool of id to randomize.

Reply:
`read_many` requires the ids to be queried, we could use `read_all`... but we would be in the same situation than querying the db... When creating demo records or anything in bulk, it could be that the vocab records are not yet there (commited or refreshed) and therefore it will fail.

Reply:
Ah right, now that vocabulaires are actually loaded when the worker starts i.e. after services setup. oh well.